tests/unit/test_filters.py

```
@@ -0,0 +1,407 @@
+Unit tests for PR filtering functionality.
+
+Tests various PR filter implementations.
+"""
+
+import unittest
+from datetime import datetime, timedelta
+from unittest.mock import Mock, patch
+
+from gh_pr.core.filters import CommentFilter as PRFilter
+# Mock other classes that don't exist
+class StateFilter: pass
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)